.github/workflows/ros2-ci.yaml

```
@@ -9,37 +9,44 @@ on:
 jobs:
 
   build-ros2:
-    runs-on: ubuntu-latest
+    runs-on: [ubuntu-latest]
```

To minimize the diff.

```suggestion
    runs-on: ubuntu-latest
```